app/models/sign_in/idme_public_key.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module SignIn
+  class IdmePublicKey
+    # rubocop:disable ThreadSafety/ClassAndModuleAttributes
+    class << self
+      attr_writer :public_keys
+    end
+    # rubocop:enable ThreadSafety/ClassAndModuleAttributes
+
+    class << self
+      attr_reader :public_keys
+    end
```

Comment:
Could you use `attr_accessor` here? Also, I wonder if we do need to worry about the thread safety 🤔 
```ruby
class << self
  attr_accessor :public_keys
end

Reply:
Or even `class_attribute` then you could  avoid the `class << self`
```ruby
class IdmePublicKey
  class_attribute :public_keys
end
```